ui/src/components/CoreConcepts/DecrementButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Button} from 'react-bootstrap';
+
+class DecrementButton extends React.PureComponent {
+    render () {
+        return (
+        <div className="d-grid gap-2">
```

Comment:
This `div` looks like part of the external layout hardcoded into the component. It should be extracted. It would be hard to reuse this component if it has some specific styles bundled into it

Reply:
Yep, it looks like some container, but it is actually a part of the button style that makes it to span all over its parent component.

Reply:
it should be controlled externally by classNames or something else